README.md

```diff
@@ -23,7 +23,7 @@ console.log(dom.window.document.querySelector("p").textContent); // "Hello world
 
 (Note that jsdom will parse the HTML you pass it just like a browser does, including implied `<html>`, `<head>`, and `<body>` tags.)
 
-The resulting object is an instance of the `JSDOM` class, which contains a number of useful properties and methods besides `window`. In general it can be used to act on the jsdom from the "outside," doing things that are not possible with the normal DOM APIs. For simple cases, where you don't need any of this functionality, we recommend a coding pattern like
+The resulting object is an instance of the `JSDOM` class, which contains a number of useful properties and methods besides `window`. In general, it can be used to act on the jsdom from the "outside," doing things that are not possible with the normal DOM APIs. For simple cases, where you don't need any of this functionality, we recommend a coding pattern like
```

Comment:
The use of a comma here is consistent with how the phrase is used elsewhere in the README:
- https://github.com/jsdom/jsdom/blame/master/README.md#L174
- https://github.com/jsdom/jsdom/blame/master/README.md#L6

Reply:
I've used blame mode as I can't figure out how to render line numbers for `.md` files otherwise!